Gemfile

```diff
@@ -9,10 +9,11 @@ gem 'solidus', github: 'solidusio/solidus', branch: branch
 # Needed to help Bundler figure out how to resolve dependencies,
 # otherwise it takes forever to resolve them.
 # See https://github.com/bundler/bundler/issues/6677
-gem 'rails', '>0.a'
+# gem 'rails', '>0.a'
```

Comment:
Can we remove this?

Reply:
Removed comments and rails version flag!